usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -0,0 +1,409 @@
+            '--delete',
+            action='store_true',
+            dest='delete',
+            default=False,
+            help='Delete the submission if it exists instead of updating it',
+        )
+
+        parser.add_argument(
+            '--test',
+            action='store_true',
+            dest='test',
+            default=False,
+            help='Runs the submission loader in test mode, sets the delete flag enabled, and uses stored data rather than pulling from a database'
+        )
+
+    def handle(self, *args, **options):
+        # Grab the data broker database connections
+        if not options['test']:
+            try:
+                db_conn = connections['data_broker']
+                db_cursor = db_conn.cursor()
+            except Exception as err:
+                self.logger.critical('Could not connect to database. Is DATA_BROKER_DATABASE_URL set?')
+                self.logger.critical(print(err))
+                return
+        else:
+            options['delete'] = True
+            db_cursor = PhonyCursor()
+
+        # Grab the submission id
+        submission_id = options['submission_id'][0]
+
+        # Verify the ID exists in the database, and grab the data
+        db_cursor.execute('SELECT * FROM submission WHERE submission_id = %s', [submission_id])
+        submission_data = dictfetchall(db_cursor)
+
+        if len(submission_data) == 0:
+            self.logger.error('Could not find submission with id ' + str(submission_id))
+            return
+        elif len(submission_data) > 1:
+            self.logger.error('Found multiple submissions with id ' + str(submission_id))
+            return
+
+        # We have a single submission, which is what we want
+        submission_data = submission_data[0]
+
+        # Create the submission data instances
+        # We load in currently available data, but the model references will
+        # stick around so we can update and save as we get more data from
+        # other tables
+
+        # First, check if we already have entries for this submission id
+        submission_attributes = None
+        update = False
+        try:
+            submission_attributes = SubmissionAttributes.objects.get(pk=submission_id)
+            if options['delete']:
+                # self.logger.info('Submission id ' + str(submission_id) + ' already exists. It will be deleted.')
+                submission_attributes.delete()
+                submission_attributes = SubmissionAttributes()
+            else:
+                # self.logger.info('Submission id ' + str(submission_id) + ' already exists. Records will be updated.')
+                update = True
+        except ObjectDoesNotExist:
+            submission_attributes = SubmissionAttributes()
+
+        # Update and save submission attributes
+        # Create our value map - specific data to load
+        value_map = {
+            'user_id': 1,
+            'update_date': timezone.now()
```

Comment:
I think @playmakervi and @alexjajabah went through and updated the `create_date` and `update_fields` to use Django's auto date fields. If the models using this dict already have that, we shouldn't need to set the create or update explicitly - Django will do it on `model.save()` for us.